testutils/util/websockets.py

```diff
@@ -39,16 +42,21 @@ async def connect():
                 self.url, extra_headers=self.headers, ssl=ssl_context
             )
 
-        attempts = 5
+        attempts = 15
+        sleep_seconds = 15
```

Comment:
@kacf the truth is: with that in here I was not ble to break it in any way. To the best of my testing this should get rid of the exception on lack of 101 being propagated.